File: Ruby/lib/Point.rb
class Point
    attr_accessor :x, :y
    def initialize (x, y)
        @x = x
        @y = y
    end
    def setPoint(x, y)
        @x = x
        @y = y
    end
    def getX()
        return @x
    end
    def getY()
        return @y
    end
    def printPoint()
        print  '[', @x , ' , ',  @y, ']'
        puts
    end
end

File: Python/src/Point.py
class Point:
    def __init__(self, x, y):
        self.__x = x
        self.__y = y

    def setPoint(self, x, y):
        self.__x = x
        self.__y = y

    def getX(self):
        return self.__x

    def getY(self):
        return self.__y

    def printPoint(self):
        print(self.__x, self.__y)
